package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prezly/uploadcare-image",
-  "version": "0.3.2-2",
+  "version": "0.3.2",
```

Comment:
This would result in some issues when we'll need to release the new preview version.
You need to either exclude the version change from this PR, or release a `preminor` preview version (`0.4.0-0`) after merging this PR.